data/Pandora.Definitions.MicrosoftGraph.Beta/Applications/Beta/ApplicationSynchronizationJob/Operation-PauseSynchronizationJob.cs

```diff
@@ -0,0 +1,24 @@
+using Pandora.Definitions.CustomTypes;
+using Pandora.Definitions.Interfaces;
+using Pandora.Definitions.MicrosoftGraph.Beta.CommonTypes;
+using System.Collections.Generic;
+using System.Net;
+using System;
+
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
```

Comment:
we'll need these around the other way, else `copywrite` may not be able to pick them up since it only checks the first few bytes